{
  try {
    s="test";
  }
 catch (  Exception e) {
  }
}
